test/karma.conf.js

```diff
@@ -31,7 +32,10 @@ module.exports = function(config) {
       module: {
         rules: [{
           test: /\.js$/,
-          loader: 'babel'
+          loader: 'babel',
+          query: {
+            presets: ['behance']
```

Comment:
The test directory doesn't have a `.babelrc`. There was one in the root of the project, but I'm removing it because when a host (be.net) consumes it, the host's webpack attempts to read the `.babelrc` and throws an error because of babel version differences.